images/capi/packer/openstack/packer.json

```diff
@@ -4,20 +4,21 @@
       "config_drive": "{{user `attach_config_drive`}}",
       "flavor": "{{user `flavor`}}",
       "floating_ip_network": "{{user `floating_ip_network`}}",
-      "image_disk_format": "raw",
```

Comment:
This contains some of the same changes as per the #6 PR and so I'd be reluctant to accept it until that one has gone through testing and merging. Let me review that one and test it first then we can revisit this one to see if it is still required.